pkg/webhook/pod/mutating/sidecarset.go

```diff
@@ -239,6 +258,9 @@ func buildSidecars(isUpdated bool, pod *corev1.Pod, oldPod *corev1.Pod, matchedS
 			}
 		}
 
+		//process imagePullSecrets
```

Comment:
consider move the code to L290 so  that new pullsecrets only affect new created pod

Reply:
good idea

Reply:
how about removing the code to L253?
if to L290, the pullsecrets will be processed repeatedly when there are more than one sidecar containers.